defaults/aws_config.yaml

```diff
@@ -8,24 +8,54 @@ instance_type_loader: 'c5.xlarge'
+  ap-south-1: # Asia Pacific (Mumbai)
+    ami_id_loader: 'ami-0c6faa5cd9cf50266' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-0f69bc5520884278e' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  eu-west-3: # Europe (Paris)
+    ami_id_loader: 'ami-0600b5654bf0d145c' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-03c476a1ca8e3ebdc' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  eu-west-2: # Europe (London)
+    ami_id_loader: 'ami-0370fbb289d8310d2' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-01b8d743224353ffe' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  eu-west-1: # EU (Ireland)
+    ami_id_loader: 'ami-042cf1bc21e30ce60' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-026e72e4e468afa7b' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  ap-northeast-2: # Asia Pacific (Seoul)
+    ami_id_loader: 'ami-071d00526e9155cbe' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-0f0646a5f59758444' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  ap-northeast-1: # Asia Pacific (Tokyo)
+    ami_id_loader: 'ami-0c25796ead2413d0c' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-0e2bf1ada70fd3f33' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  sa-east-1: # South America (São Paulo)
+    ami_id_loader: 'ami-06037dd084892cb6e' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-07ac54771249f286c' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  ca-central-1: # Canada (Central)
+    ami_id_loader: 'ami-0f4f26c6055d001d9' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-09bbc74353976b63f' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  ap-southeast-1: # Asia Pacific (Singapore)
+    ami_id_loader: 'ami-00a2cba01e3b307c1' # Loader dedicated AMI scylla-qa-loader-ami-v19
+    ami_id_monitor: 'ami-029562ad87fe1185c' # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-amd64-server-20221206 Canonical, Ubuntu, 22.04 LTS, amd64 jammy image build on 2022-12-06
+  ap-southeast-2: # Asia Pacific (Sydney)
+    ami_id_loader: 'ami-0679f687c2ab1a829' # Loader dedicated AMI scylla-qa-loader-ami-v19
```

Comment:
Why do we need all these regions?
We probably don't have quota there.

Reply:
I was under the impression we do want ability to run on any region we want.
and that we limited it just to the region we tested so far. (and by having running Jenkins workers)

You want to leave the limit in place ? to only the 6-7 region that was there before ?

Reply:
I think the general approach now is to limit to less regions, so we won't have too many AMIs and snapshots that we will pay for.